src/scenes/ProgressReports/EditForm/Steps/SubmitReportStep/ConfirmVariantDialog.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Typography,
+} from '@mui/material';
+import { Role } from '~/api/schema/schema';
+import { PromptResponseListFragment } from '~/common/fragments';
+import { useSession } from '~/components/Session';
+import { ProgressReportEditFragment } from '../../ProgressReportEdit.graphql';
+
+interface ConfirmEmptyVariantsProps {
+  report: ProgressReportEditFragment;
+  open: boolean;
+  onContinue: () => void;
+  onCancel: () => void;
+  onUpdateConfirmStatus: () => void;
+}
```

Comment:
It's good practice to forward on the props of the root component you are wrapping.
```suggestion
interface ConfirmEmptyVariantsProps extends DialogProps {
  report: ProgressReportEditFragment;
  //  open: boolean; // already inherited
  onConfirm: () => void; // matches component name better
  // onCancel: () => void; // could probably just be inherited onClose
}
```

Thinking again, maybe just using a `DialogForm` would be better/easier. This handles the title, content, and button actions for you. Here's an example:
https://github.com/SeedCompany/cord-field/blob/master/src/components/posts/DeletePost/DeletePost.tsx#L27-L42